src/main/java/ch/uzh/ifi/hase/soprafs22/controller/LobbyController.java

```diff
@@ -47,7 +48,7 @@ public class LobbyController {
     @PostMapping("/{apiVersion}/game/lobby")
     @ResponseStatus(HttpStatus.CREATED)
     @ResponseBody
-    public Map<String, Object> createLobby(@RequestHeader("token") String token, @RequestBody LobbyPostDTO lobbyPostDTO) {
+    public Map<String, Object> createLobby(@RequestHeader("token") String token, @RequestBody LobbyPostDTO lobbyPostDTO) throws PlayerNotFoundException {
```

Comment:
Why does that throw a PlayerNotFoundException? That should be catched in the lobby service and a responsestatusexception thrown

Reply:
Because we look for a player by the token, and there might not be one. PlayerNotFoundException is a checked exception so, if it wasnt caught, it would not compile.

Reply:
Fixed. Indeed, the host will always be there so I can just use that one.